MyOwnORM/MyOwnORM/CustomDbSetRepository.cs

```diff
@@ -0,0 +1,618 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM
+{
+    public class CustomDbSetRepository<T> : CustomDbSetReflection<T> where T : class
```

Comment:
why do you inherit this class of Service with reflection? 
dbset should implement interface like 'ICustomDbSetRepository' and use CustomDbSetReflection as service or helper(I think it should be service)

Reply:
I changed

Reply:
Fixed